pom.xml

```diff
@@ -113,6 +113,7 @@
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>2.9.1</version>
                 <configuration>
+                    <show>public</show>
```

Comment:
This is supposed to limit the javadocs to showing only public methods, to make that documentation relevant for API users (not implementers). TBH it didn't seem to do what it advertised, I can remove it.